sdk/storage/Azure.Storage.Files.DataLake/src/Models/DataLakePathCreateOptions.cs

```diff
@@ -57,7 +57,9 @@ public class DataLakePathCreateOptions
 
         /// <summary>
         /// Optional encryption context that can be set the file.
-        /// Encryption context is intended to store metadata that can be used to decrypt the blob.
+        /// Encryption context is file metadata that is not encrypted when stored on the file.
+        /// The primiary application of this field is to store non-encrypted data that can be used to derive the customer-provided key
```

Comment:
NIT - spelling primiary => primary

Reply:
ctrl-f and replace for this, since it's copy pasted elsewhere

Reply:
Fixed.